### DISCUSSION TITLE
Deeplinking on iOS

### DISCUSSION THREAD

**Discussion by Lascorbe** [WalletConnect/walletconnect-monorepo#2477] [Developer Support] [answered]
Hi there! I have a couple of questions regarding deeplinking on iOS:

- What should we put on the Mobile Linking's deeplink field (on cloud.walletconnect.com): `example://` or `example://wc?uri=?`
- What's the best practice to have several deeplinks for different apps which is in fact the same one (e.g. debug, staging, prod versions of the same app, they have a different bundleID and different deeplinks), is it ok to create a project for each? should that project be published or can stay as a draft?

**Comment by finessevanes** [comment, marked as answer]
>What should we put on the Mobile Linking's deeplink field (on cloud.walletconnect.com): example:// or example://wc?uri=?

[docs](https://docs.walletconnect.com/2.0/ios/guides/mobile-linking#ios-wallet-support): `example://wc?uri=?`

> What's the best practice to have several deeplinks for different apps which is in fact the same one (e.g. debug, staging, prod versions of the same app, they have a different bundleID and different deeplinks), is it ok to create a project for each? should that project be published or can stay as a draft?

I would recommend creating a new project. We only publish projects that are on prod, so it can live in Draft forever :)

**Reply by Lascorbe** [reply]
great thank you @finessevanes!
